app/services/activitypub/process_account_service.rb

```diff
@@ -6,6 +6,8 @@ class ActivityPub::ProcessAccountService < BaseService
   # Should be called with confirmed valid JSON
   # and WebFinger-resolved username and domain
   def call(username, domain, json)
+    return unless json['inbox'].present?
```

Comment:
Unintended protocol changing will affects feed receiving, so I added this check here to ensure validity.

Maybe raise exception would be more useful for debugging.